kedro-telemetry/tests/test_plugin.py

```diff
@@ -9,15 +9,14 @@
 from kedro.io import DataCatalog, MemoryDataSet
 from kedro.pipeline import node
 from kedro.pipeline.modular_pipeline import pipeline as modular_pipeline
-from pytest import fixture
-
 from kedro_telemetry import __version__ as TELEMETRY_VERSION
 from kedro_telemetry.plugin import (
     KedroTelemetryCLIHooks,
     KedroTelemetryProjectHooks,
     _check_for_telemetry_consent,
     _confirm_consent,
 )
+from pytest import fixture
```

Comment:
The linter here seems to think that `kedro_telemetry` is a third-party import (which it shouldn't).

Reply:
@sbrugman Good spot, but I would leave it for now as I am trying to introduce `ruff` for `kedro` , the next step will be `kedro_plugins` and we can add this as a known first-party library.

https://github.com/kedro-org/kedro/pull/2634

Reply:
Awesome, feel free to tag me as reviewer (using `ruff` everywhere now)

Reply:
Managed to fix this issue.